pyproject.toml

```diff
@@ -1,6 +1,3 @@
-[tool.black]
-line-length = 99
-
```

Comment:
Maybe it's worth keeping this here, so black can still be used if that's what the user has installed??  Or do we say that black is no longer supported?  I guess that's technically true because we now strictly follow ruff formatting.

Reply:
I think black is no longer supported.

Reply:
OK

Reply:
I've made a similar change on a local project and yes, I found it's really one or the other - even the trivially minor differences between the two formatters means you can't really keep both enabled.